CI/freebsd/01_install_dependencies.sh

```diff
@@ -79,6 +83,7 @@ print_usage() {
             "-h, --help                     : Print this help\n" \
             "-q, --quiet                    : Suppress most build process output\n" \
             "-v, --verbose                  : Enable more verbose build process output\n"
+            "--disable-pipewire             : Disable building with Pipewire support (default: off)\n"
```

Comment:
Nit: Pipewire -> PipeWire

Reply:
`Pipewire` spelling was inherited from Linux config, so I had to fix there as well.